### DISCUSSION TITLE
Is there a way to update meta tags in the head while using hx-boost

### DISCUSSION THREAD

**Discussion by Tobi-De** [bigskysoftware/htmx#1125] [Q&A] [answered]
Hello, I hope everyone is ok,
I'm trying to use hx-boost in one of my projects, it works fine but there is a small problem.
I am using django and in some of my templates, I create a block using django's template language to update some meta tags in my html head. Without hx-boost it works fine, but when I add it, since it only replaces the body, the `head` tag content in my html is not updated, is there any way around this?
Example:

base template
```html
<head>
   ...
    {% block header_meta %}
        <meta name="description" content="My description here">
    {% block header_meta %}
    <link rel="stylesheet" href="{% static 'index.css' %}">
    ...
```

another template somewhere 

```
    {% block header_meta %}
 <meta name="description" content="updated description">
    {% block header_meta %}
```

**Comment by Tobi-De** [comment, marked as answer]
I found happiness with https://htmx.org/extensions/head-support/

**Comment by RudyAnconi** [comment]
Well, the head-support extension is adding the new metas instead of replacing them.

If you have the code:
```
<meta name='description' content='Document'>
```
and send on the response:
```
<meta name='description' content='Document loaded'>
```
The extension will not evaluate the name of the meta and instead of replacing the tag, will append at the bottom.
